phpBB/docs/events.md

```diff
@@ -2227,6 +2227,14 @@ viewforum_body_topic_row_prepend
 * Since: 3.1.7-RC1
 * Purpose: Add content at the end of the topic list item.
 
+viewforum_body_topicrow_row_before
```

Comment:
Shouldn't it be **viewforum_body_topicrow_before** ¿?

Applies to all of this files.